tests/test_complexity.py

```diff
@@ -104,3 +101,52 @@ def test_ft_methods_complexity(self, dt_id, ft_name, exp_value,
             assert value[0] is exp_value
         else:
             assert np.allclose(value, exp_value, equal_nan=True)
+
+    @pytest.mark.parametrize(
+        "dt_id, exp_value, precompute",
+        [
+            ###################
+            # Mixed data
+            ###################
+            (0, [
+                1.0, 0.0, 0.76, 0.66, 0.24, 1.0, 0.48, 0.22, 0.02, 0.09090909
+            ], False),
+            (0, [
+                1.0, 0.0, 0.76, 0.66, 0.24, 1.0, 0.48, 0.22, 0.02, 0.09090909
+            ], True),
+            ###################
```

Comment:
These commentaries are related to the inner-metafeature group groups for the Categorical test data. Because that dataset is large, enabling those tests for every metafeature group would cost way too much for no much real gain. So, after I ran those tests once (to verify that everything was OK,) I disabled them but kept the code just for future reference.